test/CMakeLists.txt

```diff
@@ -37,8 +37,7 @@ install(FILES ${test_data}
 list(APPEND test_input
   ${PROJECT_SOURCE_DIR}/test/testinput/check_yaml_keys_ref.yaml
   ${PROJECT_SOURCE_DIR}/test/testinput/check_yaml_keys_test.yaml
-  ${PROJECT_SOURCE_DIR}/test/testinput/generate_yaml.yaml
-  ${PROJECT_SOURCE_DIR}/test/testinput/generate_yaml_include.yaml
+  ${PROJECT_SOURCE_DIR}/test/testinput/genYAML_example.yaml
```

Comment:
Remove that line, that file doesn't exist and isn't needed. 
or
remove the test and script that generates it and add the template yaml?
Your choice!

Reply:
Yep, thanks, that is an erroneous add, removed.